testcafe/tests/photos/create_full_album_scenario.js

```diff
@@ -1,96 +1,211 @@
+  await photoAlbumPage.renameAlbum(FEATURE_PREFIX, `New_${FEATURE_PREFIX}`, {
     exitWithEnter: true
   })
-  await photoAlbumPage.checkAlbumPage(`New_${ALBUM_DATE_TIME}`, 3)
+  await photoAlbumPage.checkAlbumPage(`New_${FEATURE_PREFIX}`, 3)
   console.groupEnd()
 })
 
-test('Go to New_ALBUM_DATE_TIME, and rename it (exit by clicking away)', async () => {
-  console.group(
-    `↳ ℹ️  Go to New_${ALBUM_DATE_TIME}, and rename it (exit by clicking away)`
-  )
+test(TEST_RENAME_ALBUM2, async () => {
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_RENAME_ALBUM2}`)
   await timelinePage.goToAlbums()
-  await photoAlbumsPage.goToAlbum(`New_${ALBUM_DATE_TIME}`)
-  await photoAlbumPage.checkAlbumPage(`New_${ALBUM_DATE_TIME}`, 3)
+  await photoAlbumsPage.goToAlbum(`New_${FEATURE_PREFIX}`)
+  await photoAlbumPage.checkAlbumPage(`New_${FEATURE_PREFIX}`, 3)
   await photoAlbumPage.renameAlbum(
-    `New_${ALBUM_DATE_TIME}`,
-    `New2_${ALBUM_DATE_TIME}`,
+    `New_${FEATURE_PREFIX}`,
+    `New2_${FEATURE_PREFIX}`,
     { exitWithEnter: false }
   )
-  await photoAlbumPage.checkAlbumPage(`New2_${ALBUM_DATE_TIME}`, 3)
+  await photoAlbumPage.checkAlbumPage(`New2_${FEATURE_PREFIX}`, 3)
   console.groupEnd()
 })
 
-test('Go to New2_ALBUM_DATE_TIME, and add 2 more photos', async () => {
-  console.group(`↳ ℹ️  Go to New2_${ALBUM_DATE_TIME}, and add 2 more photos`)
+test(TEST_ADD_MORE_PHOTOS, async () => {
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_ADD_MORE_PHOTOS}`)
   await timelinePage.goToAlbums()
-  await photoAlbumsPage.goToAlbum(`New2_${ALBUM_DATE_TIME}`)
-  await photoAlbumPage.addPhotosToAlbum(`New2_${ALBUM_DATE_TIME}`, 3, 2)
+  await photoAlbumsPage.goToAlbum(`New2_${FEATURE_PREFIX}`)
+  await photoAlbumPage.addPhotosToAlbum(`New2_${FEATURE_PREFIX}`, 3, 2)
   await photoAlbumPage.backToAlbumsList()
-  await photoAlbumsPage.isAlbumExistsAndVisible(`New2_${ALBUM_DATE_TIME}`, 5)
+  await photoAlbumsPage.isAlbumExistsAndVisible(`New2_${FEATURE_PREFIX}`, 5)
   console.groupEnd()
 })
 
-test('Go to New2_ALBUM_DATE_TIME, and remove the 1st photos', async () => {
-  console.group(
-    `↳ ℹ️  Go to New2_${ALBUM_DATE_TIME}, and remove the 1st photos`
+test(TEST_ADD_EXISTING_PHOTOS, async t => {
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_ADD_EXISTING_PHOTOS}`)
+  await timelinePage.selectPhotos(1)
+  await timelinePage.checkCozyBarOnTimeline()
+  await t.click(selectors.btnAddToAlbumCozySelectionBar)
+
+  await t.expect(selectors.spinner.exists).notOk('Spinner still spinning')
+  await checkAllImagesExists()
+
+  await t.fixtureCtx.vr.takeScreenshotAndUpload({
+    screenshotPath: `${FEATURE_PREFIX}/${TEST_ADD_EXISTING_PHOTOS}-1`,
+    withMask: maskPhotosCluster
+  })
+  await t.click(selectors.album(`New2_${FEATURE_PREFIX}`))
+
+  //Wait for toast alert to disapear before taking screenshot
+  await checkToastAppearsAndDisappears(
+    'This album already contains this photo.'
   )
+  //go to album to get the screenshot
```

Comment:
I think this kind of comments is not very useful if the code is clear which is the case here : 
- `timelinePage.goToAlbums`
- `photoAlbumsPage.goToAlbum`
- `t.fixtureCtx.vr.takeScreenshotAndUpload`